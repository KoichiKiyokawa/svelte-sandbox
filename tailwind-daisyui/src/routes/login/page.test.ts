import { render } from '@testing-library/svelte';
import { describe, it, expect } from 'vitest';
import Page from './+page.svelte';

describe('login page', () => {
	it('to match snapshot', () => {
		const { container } = render(Page);
		expect(container).toMatchSnapshot();
	});
});
