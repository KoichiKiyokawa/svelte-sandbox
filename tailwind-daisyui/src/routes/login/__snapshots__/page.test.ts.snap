// Vitest Snapshot v1

exports[`login page > to match snapshot 1`] = `
<body>
  <div>
    <div
      class="card max-w-screen-md mx-auto bg-base-300 shadow-xl mt-4"
    >
      <div
        class="card-body"
      >
        <h2
          class="card-title"
        >
          Login
        </h2>
         
        <form>
          <div
            class="form-control w-full "
          >
            <label
              class="label"
              for="email"
            >
              <span
                class="label-text"
              >
                Email:
              </span>
            </label>
             
            <input
              class="input input-bordered w-full"
              id="email"
              placeholder="foo@example.com"
              type="email"
            />
          </div>
          <!--&lt;Index&gt;-->
           
          <div
            class="form-control w-full mt-4"
          >
            <label
              class="label"
              for="password"
            >
              <span
                class="label-text"
              >
                Password:
              </span>
            </label>
             
            <input
              class="input input-bordered w-full"
              id="password"
              placeholder=""
              type="password"
            />
          </div>
          <!--&lt;Index&gt;-->
           
          <button
            class="btn btn-primary mt-8"
          >
            Login
          </button>
        </form>
      </div>
    </div>
    <!--&lt;+page&gt;-->
  </div>
</body>
`;
